src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1009,6 +1009,53 @@ public function testMirrorCopiesRelativeLinkedContents()
         $this->assertEquals('\\' === DIRECTORY_SEPARATOR ? realpath($sourcePath.'\nested') : 'nested', readlink($targetPath.DIRECTORY_SEPARATOR.'link1'));
     }
 
+    public function testMirrorContentsWithSameNameAsSourceOrTargetWithoutDeleteOption()
+    {
+        $sourcePath = $this->workspace.DIRECTORY_SEPARATOR.'source'.DIRECTORY_SEPARATOR;
+
+        $oldPath = getcwd();
```

Comment:
We do not need the `chdir()` call after the assertions at the end of both methods. Therefore, we can also removed the `$oldPath` variable.